cve_bin_tool/validator.py

```diff
@@ -7,15 +7,17 @@
 
 from cve_bin_tool.log import LOGGER
 
-SCHEMA_LOCATION_DEFAULT = "./cve_bin_tool/schemas/"
-
 
 def _validate_xml(filename, xsd_file):
 
-    theschema = xmlschema.XMLSchema(os.path.join(SCHEMA_LOCATION_DEFAULT, xsd_file))
-    LOGGER.debug(f"Validate {filename} against {theschema}")
+    # Root folder where schemas are present
+    root = os.path.dirname(os.path.abspath(__file__))
```

Comment:
We should probably use pathlib here, since we've been switching to it elsewhere.
https://docs.python.org/3/library/pathlib.html